### ISSUE TITLE
Cache doesn't clear up when using bypassCache option

### ISSUE THREAD

**Issue by gnuwilliam** [cloudflare/wrangler#1399]
#### Environment

* operating system: `macOS`
* output of `rustc -V`:`rustc 1.33.0 (2aa4c46cf 2019-02-28)`
* output of `node -v`: `v12.16.1`
* output of `wrangler -V`: `👷 ✨  wrangler 1.9.2`
* contents of `wrangler.toml`: `type: webpack` - using a standard Gatsby template

#### Steps to reproduce

- Run `wrangler publish` with `bypassCache: true` or set the `edgeTTL` to a specific value and send a new build
- Access the build

#### What did you expect to see?

- The new version of the build being served

#### What did you see instead?

- The older version of the build (have to manually empty the browser cache, on both desktop and mobile - Chrome latest version, to see the new version)

---

Basically, what's happening is that when I send a new build using the cache options available, instead of getting a fresh version of the build (it's a Gatsby static website), I see the older version, and have to keep cleaning the cache on the client side to see the new content. What I'd expect is that when I send a new version of the static content, it'll serve that new version, and cache it (say for 24h or 1 week).

I'd like to ask if there's any additional step that I didn't take, or any extra documentation on caching that would help me solve this issue, maybe a manual purge on the worker or something? I'd be glad to share more information if needed.

Thank you!

**Comment by EverlastingBugstopper** [comment]
Hey @gnuwilliam - it sounds like you're talking about a Workers Site, is that correct? If so, I'll move this issue to the [correct repo](https://github.com/cloudflare/kv-asset-handler) and we can take a look there. I'd also just real quick make sure that you're using the latest `kv-asset-handler` in your `package.json`.

**Comment by EverlastingBugstopper** [comment]
@shagamemnon - do you think this is fixed w/the support for etag in `kv-asset-handler`?

**Comment by shagamemnon** [comment]
@EverlastingBugstopper - `etag` solves this problem if `bypassCache: false` yes. If `bypassCache: true`, then we won't set an `etag` version identifier on the response, so the browser won't have any comparison context (in other words, this problem will occur)

@gnuwilliam - are you setting a browserTTL? If you are _not_ setting a browserTTL, then this behavior is certainly odd and should be investigated further. If you are setting a browserTTL, keeping `bypassCache: false` in place (e.g. `{ bypassCache: false, edgeTTL: 86400, browserTTL: 3600 }`) will actually resolve this problem once https://github.com/cloudflare/kv-asset-handler reaches its next release when `kv-asset-handler` will start supporting [ETags](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/ETag)
